lightning-invoice/src/utils.rs

```diff
@@ -922,62 +919,6 @@ mod test {
 		);
 	}
 
-	#[test]
-	#[cfg(feature = "std")]
-	fn test_multi_node_forwarding_info_not_assigned_channel_excluded_from_hints() {
```

Comment:
This test required that the `Channel::counterparty_forwarding_info` field wasn't assigned until the `ChannelUpdate` msg. The `Channel::counterparty_forwarding_info` field will now be assigned as soon as it's accepted by both parties.